starter_code/index.html

```diff
@@ -10,6 +10,8 @@
       <!-- Link JQuery here -->
 
       <!-- Link script.js here -->
+      <script src= "script.js"></script>
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
you should switch the order of the two. if `script.js` is relying on `jquery` you'll have to switch em around